setup.sh

```diff
@@ -86,14 +86,14 @@ fi
 
 # Adjust permissions for ossec-wui
 echo "Enter your web server user name (e.g. apache, www, nobody, www-data, ...)"
-read GROUP
-OSSEC=`grep ossec /etc/group`
-NEWLINE=$OSSEC$GROUP
-sed "s/$OSSEC/$NEWLINE/" -i /etc/group
-echo "Enter your OSSEC install directory path (e.g. /var/ossec)"
-read INSTALL
-chmod 770 $INSTALL/tmp/
-chgrp $GROUP $INSTALL/tmp/
+read HTTPDUSER
+OSSEC=`grep ^ossec: /etc/group`
+if ! (echo $OSSEC | grep -w $HTTPDUSER) > /dev/null 2>&1; then
+    NEWLINE="$OSSEC,$HTTPDUSER"
+    NEWLINE=`echo $NEWLINE | sed -e 's/:,/:/'`
+    sed "s/$OSSEC/$NEWLINE/" -i /etc/group
```

Comment:
On Fri, 8 May 2015, ddpbsd wrote:

> Date: Fri, 08 May 2015 06:25:50 -0700
> From: ddpbsd <notifications@github.com>
> Reply-To: ossec/ossec-wui
>     <reply+0063a4d3b181531c6ee10f78e601f8e5c3ccf4d31d3612f392cf0000000111647b5
>     e92a170ce01c8d4af@reply.github.com>
> To: ossec/ossec-wui <ossec-wui@noreply.github.com>
> Cc: Antonio Querubin <tony@lavanauts.org>
> Subject: Re: [ossec-wui] Fixup setup.sh group modification and add selinux
>     info. (#10)
> 
>> @@ -86,14 +86,14 @@ fi
>>
>>  # Adjust permissions for ossec-wui
>>  echo "Enter your web server user name (e.g. apache, www, nobody, www-data, ...)"
>> -read GROUP
>> -OSSEC=`grep ossec /etc/group`
>> -NEWLINE=$OSSEC$GROUP
>> -sed "s/$OSSEC/$NEWLINE/" -i /etc/group
>> -echo "Enter your OSSEC install directory path (e.g. /var/ossec)"
>> -read INSTALL
>> -chmod 770 $INSTALL/tmp/
>> -chgrp $GROUP $INSTALL/tmp/
>> +read HTTPDUSER
>> +OSSEC=`grep ^ossec: /etc/group`
>> +if ! (echo $OSSEC | grep -w $HTTPDUSER) > /dev/null 2>&1; then
>> +    NEWLINE="$OSSEC,$HTTPDUSER"
>> +    NEWLINE=`echo $NEWLINE | sed -e 's/:,/:/'`
>> +    sed "s/$OSSEC/$NEWLINE/" -i /etc/group
>
> The "-i" flag isn't available on all seds.

That part wasn't changed so if it was broken before on a particular OS, 
then I suppose it's still broken.

Antonio Querubin
e-mail:  tony@lavanauts.org
xmpp:  antonioquerubin@gmail.com